fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/impl/LookupDimension.java

```diff
@@ -51,6 +59,32 @@ public boolean equals(Object o) {
             Objects.equals(namespaces, that.namespaces);
     }
 
+    /**
+     * Build an extraction function model object.
+     *
+     * @return  Take the internal namespaces and construct a model object for the extraction functions.
+     */
+    @JsonIgnore
+    public Optional<ExtractionFunction> getExtractionFunction() {
+
+        List<ExtractionFunction> extractionFunctions = getNamespaces().stream()
+                .map(
+                        namespace -> new LookupExtractionFunction(
+                                new NamespaceLookup(namespace),
+                                false,
+                                "Unknown " + namespace,
```

Comment:
While we are at it, we can update this message to `__unsupportedValues` or something like it since this value will be shown to the user. `Unknown something__something` does not make much sense for the UI to show I guess.